pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -43,6 +44,26 @@
   USE_USB = False
 
 
+def need_iswap_parked(method: Callable):
+  """Ensure that the iSWAP is in parked position before running command.
+
+  If the iSWAP is not parked, it get's parked before running the command.
+  """
+
+  @functools.wraps(method)
+  def wrapper(self, *args, **kwargs):
+    if self.iswap_parked:
+      result = method(self, *args, **kwargs)
+      self.iswap_parked = False
+    else:
+      self.park_iswap()
+      self.iswap_parked = True
```

Comment:
Similarly, `self.iswap_parked = True` should be set in `park_iswap`.

(Right now if the user would call `park_iswap()` before calling a `@need_iswap_parked` method, we would send the park command twice.)